dbt_copilot_helper/commands/conduit.py

```diff
@@ -88,14 +88,29 @@ def get_addon_type(app: Application, env: str, addon_name: str) -> str:
     return addon_type
 
 
-def get_or_create_task_name(app: Application, env: str, addon_name: str) -> str:
+def get_parameter_name(
+    app: Application, env: str, addon_type: str, addon_name: str, write: bool, admin: bool
+) -> str:
+    if addon_type == "postgres":
+        permission = "READ_ONLY"
+        if admin:
+            permission = "ADMIN"
+        elif write:
+            permission = "WRITE"
```

Comment:
If the cli option was "--access write" for example, this could be tidied up into a dictionary.

Reply:
Or even just capitalized.